DEVELOPERS.md

```diff
@@ -0,0 +1,17 @@
+## Set up for local development
+
+1. `pnpm i`
+2. `pnpm run build`
+3. `pnpm run bundle`
+
+When developing locally, configure your provider in the client (e.g., VS Code) by using the path to the bundled .js file:
+
+```json
+"openctx.providers": {
+    "file:///<path/to/js/bundle>/bundle.js": true,
+}
+```
+
+## Debug
+
+To see console log statements of your provider, open the vscode developer tools: [`cmd + Shift + P`] > "Toggle Developer Tools".
```

Comment:
this is probably okay, but just wondering if there's a better way to show debug info, maybe in Cody's output channel. @keegancsmith do you know if it's possible?

Reply:
There is not currently, but that would be a good improvement for us to add.